client/src/templates/Challenges/redux/create-question-epic.js

```diff
@@ -50,6 +50,41 @@ function createQuestionEpic(action$, state$, { window }) {
         block,
         helpCategory
       } = challengeMetaSelector(state);
+      if (
+        challengeFiles?.some(file => file.editableRegionBoundaries.length > 0)
+      ) {
+        const editableRegionStrings = fileExtension => {
+          const lineBreak = '\n';
+          const multiLineStart = '/*';
+          const multiLineEnd = '*/';
+          const htmlStart = '<!---';
+          const htmlEnd = '--->';
+          const editableRegionStr = 'User Editable Region';
+          const startComment =
+            fileExtension === 'html' ? htmlStart : multiLineStart;
+          const endComment = fileExtension === 'html' ? htmlEnd : multiLineEnd;
+          const editableRegionComments = {
+            comment: `${lineBreak}${startComment} ${editableRegionStr} ${endComment}${lineBreak}`
+          };
+          return { lineBreak, editableRegionComments };
```

Comment:
```suggestion
          const startComment = fileExtension === 'html' ? '<!--' : '/*';
          const endComment = fileExtension === 'html' ? '-->' : '*/';
          return `\n${startComment} 'User Editable Region' ${endComment}\n`;
```
A more concise version like this should be easier to read.  Also, the html comments were `<!---` instead of `<!--`, so I changed that.